src/updateNode.js

```diff
@@ -276,7 +276,7 @@ export default function updateNode (part, node, oldNode, context, forceUpdate, i
     return updateComponentNode(part, node, oldNode, context, forceUpdate, isSvgPart);
   } else if (isTagNode(node)) {
     return updateTagNode(part, node, oldNode, context, forceUpdate, isSvgPart);
-  } else if (isPrimitiveNode(node) && (node !== oldNode || forceUpdate)) {
+  } else if (isPrimitiveNode(node) && ('' + node !== oldNode || forceUpdate)) {
```

Comment:
If the nodes are number this will always fail. We need to convert the old node as well to string. Also, we should check if the old node is also primitiveNode.

Reply:
Yes makes changes, did those checks and also made it little cleaner to understand